baseline/client/tests/n-back.test.js

```diff
@@ -77,6 +84,43 @@ describe("n-back", () => {
         expect(complete).toBe(true);
     });
 
+    it("evaluated n-back trials should match spec", () => {
+        const setNums = [1, 2];
+        setNums.forEach(setNum => {
```

Comment:
Minor nit: With this pattern when a test fails there's no way to know what set it failed on. Given that there are only two it's not a huge deal, but it would be better to use [describe.each](https://jestjs.io/docs/api#describeeachtablename-fn-timeout) here.

Reply:
Fixed!